definitions/checks/candlepin/validate_candlepin_db.rb

```diff
@@ -0,0 +1,20 @@
+module Checks::Candlepin
+  class ValidateCandlepinDb < ForemanMaintain::Check
+    metadata do
+      description 'Check to validate candlepin database'
+      tags :pre_upgrade
+      label :validate_candlepin_db
+
+      confine do
+        feature(:candlepin) &&
+          feature(:candlepin_database).check_option_using_cpdb_help('validate') != 0
```

Comment:
I would recommend hiding the implementation detail, that we're checking help for this output, by calling something like `feature(:candlpin_database).validate_available?` and letting this method decide how we determine that (which in this case would be calling the `check_option_using_cpdb_help('validate')` method there.